lib/multi-acc-new-eks-mixed-observability-pattern/getSSMSecureString.ts

```diff
@@ -0,0 +1,34 @@
+import { SSM } from 'aws-sdk';
+
+// function to get SSM Parameterstor Securestring value
+export async function getSSMSecureString(parameterName: string, region: string): Promise<string> {  
```

Comment:
Why is this required? Cant we use External secrets to read SSM?

Reply:
This is used to get SSM parameter 
/cdk-accelerator/cdk-context for CodePipeline
/cdk-accelerator/pipeline-git-info for CodePipeline source repo
/cdk-accelerator/amg-info which holds AMG info like IAMRoleARN that are required to create cross account IAM role 

As these parameter values are used outside of EKS cluster, I am using SDK to get SSM value instead of External secrets.